.github/actions/action-npm-release/action.yml

```
@@ -0,0 +1,72 @@
+inputs:
+  node-version:
+    required: false
+    default: "22.11.0"
+    type: string
+    description: node version
+  app-path:
+    required: true
+    type: string
+    description: path where the package.json is located
+  releaseVersion:
+        type: choice
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 8

(indentation)